components/services/bat_ads/bat_ads_client_mojo_bridge.cc

```diff
@@ -265,6 +265,18 @@ void BatAdsClientMojoBridge::ShowScheduledCaptchaNotification(
   bat_ads_client_->ShowScheduledCaptchaNotification(payment_id, captcha_id);
 }
 
+#if BUILDFLAG(IS_ANDROID)
+void BatAdsClientMojoBridge::ShowScheduledCaptchaAndroid(
+    const std::string& payment_id,
+    const std::string& captcha_id) {
+  if (!connected()) {
```

Comment:
Nit: to save 2 lines here, coud this be:
```
if (connected()) {
  bat_ads_client_->ShowScheduledCaptchaAndroid(payment_id, captcha_id);
}
```
? 🙂

Reply:
This function should be removed based on comment so resolving this.